packages/app/src/types.ts

```diff
@@ -40,6 +40,12 @@ export type ImageComponentPlugin = Plugin & {
     presets: { [key: string]: any };
 };
 
+/**
+ * Enables registering new routes on which you can render your own views.
```

Comment:
Just `Enables registering new routes.` is enough here.

Reply:
This change has been effected.

Reply:
Great! @drexlar5